Tests/Unit Tests/JSON Validator/TWTRandomizedTestCase+TWTJSONSchemaTestDirectories.h

```diff
@@ -0,0 +1,25 @@
+//
+//  TWTRandomizedTestCase+TWTJSONSchemaTestDirectories.h
+//  TWTValidation
+//
+//  Created by Jill Cohen on 12/14/15.
+//  Copyright © 2015 Ticketmaster Entertainment, Inc. All rights reserved.
+//
+
+#import "TWTRandomizedTestCase.h"
+
+
+// Converts the preprocessor macros for test directory paths into strings
+#define STRING_FROM_SYMBOL(x) #x
+#define STRING_FROM_MACRO(x) STRING_FROM_SYMBOL(x)
+
+
+@interface TWTRandomizedTestCase (TWTJSONSchemaTestDirectories)
+
++ (NSString *)twt_pathForTestSuite;
+
++ (NSString *)twt_pathForDraft4TestSuite;
+
++ (NSString *)twt_pathForCustomTests;
```

Comment:
Might be okay to do these as functions? If you don’t want to, can we put the words JSON schema in here, since they’re being pulled in for all TWTRandomizedTestCase subclasses.